package-lock.json

```diff
@@ -0,0 +1,19940 @@
+{
```

Comment:
We use yarn, would you mind deleting this file and updating yarn.lock

Reply:
Done, my apologies!